api/desecapi/management/commands/chores.py

```diff
@@ -46,7 +48,44 @@ def update_healthcheck_timestamp():
             serializer.save()
         print(f'TXT {name} updated to {content}')
 
+    @staticmethod
+    def alerting_healthcheck():
+        name = 'external-timestamp.desec.test'
+        try:
+            models.Domain.objects.get(name=name)
+        except models.Domain.DoesNotExist:
+            print(f'{name} zone is not configured; skipping alerting health check')
+            return
+
+        now = time.time()
+        timestamps = []
+        query = dns.message.make_query(f'{name}.', dns.rdatatype.TXT)
+        server = 'ns1.desec.io'
+        response = None
+        try:
+            response = dns.query.tcp(query, server, timeout=5)
+            for rrset in response.answer:
```

Comment:
easier: `response.find_rrset(dns.message.ANSWER, qname, dns.rdataclass.IN, qtype)`